.github/workflows/dependencies.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
-        python-version: 3.8
+        python-version: 3.9
```

Comment:
the lowest version pandas to have `__dataframe_consortium_standard__` is 2.1, which only support python 3.9+

Reply:
I never know how reliable this is, yet from [pypi stats](https://pypistats.org/packages/scikit-lego) it seems that the majority of downloads comes from 3.7 and 3.8 🫥

Reply:
ah interesting - OK, keeping 3.8 then, we can work around that with a helper function, and only use the `__dataframe_consortium_standard__` for 3.9+

timeline-wise, we're aiming for Feburary to make the first non-beta release